contents/blog/posthog-vs-amplitude.md

```diff
@@ -0,0 +1,420 @@
+---
+date: 2022-05-09
+title: PostHog vs Amplitude
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+featuredImage: ../images/blog/posthog-engineering-blog.png
+featuredImageType: full
+author: ["andy-vandervell", "joe-martin"]
+categories: ["Guides", "Open source"]
+---
+
+PostHog and Amplitude are both product analytics tools. They tell you how people are using your product, not just where they come from, and help you find ways to make your product better.
+
+But we think PostHog is a more powerful tool for building better products. Not only does offer it more features and tools for engineers to understand their users, it's an open platform that you can self-host and customize to your needs.
+
+**Contents:**
+
+- [How is PostHog different to Amplitude?](#how-is-posthog-different-from-amplitude)
+- [PostHog vs Amplitude feature comprison](#feature-comparison)
+- [Integrations and data sources](#integrations-and-data-sources)
+- [Strengths of PostHog](#strengths-of-posthog)
+- [Strengths of Amplitude](#strengths-of-amplitude)
+- [Coming soon to PostHog](#coming-soon-to-posthog)
+
+## How is PostHog different from Amplitude?
+
+Let's start with the core differences. Over 9,000 companies already use PostHog, and many have switched from Amplitude. Here are a few reasons why.
+
+### 1. It's open source
+Our MIT License isn’t just for show. You can access [our source code](https://github.com/PostHog/posthog), raise your own issues and PRs, and use it to [build your own plugins](/docs/plugins/build) or even add extra functionality. You also benefit from the work of other teams who build their own plugins. And we're not just an open-source tool; we're an open-source company. Our [company handbook](/handbook) is open to everyone, as is [how we pay people](/handbook/people/compensation).
+
+### 2. It's built for engineers
+Unlike Amplitude, PostHog is built for software developers. PostHog autocaptures events, so you don't need to waste time instrumenting events every time you update your app or website. Our pricing is transparent, and we're entirely self-serve. You can setup PostHog and start paying us without ever speaking to a sales rep, but we have an [awesome customer success team](/signup/self-host/get-in-touch?plan=scale&demo=scale#demo) if you have questions.
+
+### 3. You control your data
```

Comment:
I feel this should be number 1 in the list. I feel this is more generally applicable than the first one.